packages/scikit-image/patches/unwrap.patch

```diff
@@ -0,0 +1,114 @@
+diff --git a/skimage/restoration/_unwrap_2d.pyx b/skimage/restoration/_unwrap_2d.pyx
+index ad0dfa2280..8e2d6ee3ae 100644
+--- a/skimage/restoration/_unwrap_2d.pyx
++++ b/skimage/restoration/_unwrap_2d.pyx
+@@ -3,13 +3,17 @@
+ # cython: nonecheck=False
+ # cython: wraparound=False
+ 
+-cdef extern from *:
+-    void unwrap2D(double* wrapped_image,
+-                  double* unwrapped_image,
+-                  unsigned char* input_mask,
+-                  int image_width, int image_height,
+-                  int wrap_around_x, int wrap_around_y,
+-                  char use_seed, unsigned int seed) nogil
++
++cdef extern from "unwrap_2d_ljmu.h":
++    void unwrap2D(
++            double *wrapped_image,
++            double *UnwrappedImage,
++            unsigned char *input_mask,
++            int image_width, int image_height,
++            int wrap_around_x, int wrap_around_y,
++            char use_seed, unsigned int seed
++            ) nogil
++
+ 
+ def unwrap_2d(double[:, ::1] image,
+               unsigned char[:, ::1] mask,
+diff --git a/skimage/restoration/_unwrap_3d.pyx b/skimage/restoration/_unwrap_3d.pyx
+index eac50e70fb..2c9bbfbb1d 100644
+--- a/skimage/restoration/_unwrap_3d.pyx
++++ b/skimage/restoration/_unwrap_3d.pyx
+@@ -3,13 +3,18 @@
+ # cython: nonecheck=False
+ # cython: wraparound=False
+ 
+-cdef extern from *:
+-  void unwrap3D(double* wrapped_volume,
+-                double* unwrapped_volume,
+-                unsigned char* input_mask,
+-                int image_width, int image_height, int volume_depth,
+-                int wrap_around_x, int wrap_around_y, int wrap_around_z,
+-                char use_seed, unsigned int seed) nogil
++
++cdef extern from "unwrap_3d_ljmu.h":
```

Comment:
Is it something that could be contributed upstream?

Where are they importing it from if they don't have this header file?

Reply:
This patch comes directly from a PR upstream.

Reply:
Then please mention it as a comment inside the patch